### PR TITLE
Create fix_bert_twitter.py

### PR DESCRIPTION
**Pull request by akiyama-ai**

### CODE REVIEW

File: fix_bert_twitter.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+            'ids': torch.LongTensor(ids),
+            'mask': torch.LongTensor(mask),
+            'labels': torch.LongTensor(self.y[index])
+        }
+
+
+class BERTClass(torch.nn.Module):
+    def __init__(self, bert_model_file, drop_rate, otuput_size, config_file):
+        super().__init__()
+        self.bert = BertForPreTraining.from_pretrained(bert_model_file, from_tf=True, config=config_file)
```
Comment (akiyama-ai) [inline fix_bert_twitter.py:62]: self.bert = BertForPreTraining.from_pretrained(bert_model_file, from_tf=True, config=config_file, output_hidden_states=True) 
output_hidden_states=Trueをつけることで各層の隠れ層が取得できます。

File: fix_bert_twitter.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        self.drop = torch.nn.Dropout(drop_rate)
+        self.fc = torch.nn.Linear(32000, otuput_size)
+
+    def forward(self, ids, mask):
+        out, _ = self.bert(ids, attention_mask=mask)
+        out = self.fc(self.drop(out[:,MAX_LEN-1]))
```
Comment (akiyama-ai) [inline fix_bert_twitter.py:68]: [CLS]トークンを使う場合はout = self.fc(self.drop(out[:,0]))となります。

File: fix_bert_twitter.py [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+class BERTClass(torch.nn.Module):
+    def __init__(self, bert_model_file, drop_rate, otuput_size, config_file):
+        super().__init__()
+        self.bert = BertForPreTraining.from_pretrained(bert_model_file, from_tf=True, config=config_file)
+        self.drop = torch.nn.Dropout(drop_rate)
+        self.fc = torch.nn.Linear(32000, otuput_size)
+
+    def forward(self, ids, mask):
+        out, _ = self.bert(ids, attention_mask=mask)
```
Comment (akiyama-ai) [inline fix_bert_twitter.py:67]: outputs = self.bert(ids, attention_mask=mask)
outputs["hidden_states"][-1]で最終レイヤの隠れ層取り出し

File: fix_bert_twitter.py [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    
+    dataloader_train = DataLoader(dataset_train, batch_size=BATCH_SIZE, shuffle=False)
+    dataloader_dev = DataLoader(dataset_dev, batch_size=BATCH_SIZE, shuffle=False)
+    dataloader_test = DataLoader(dataset_test, batch_size=BATCH_SIZE, shuffle=False)
+
+    print(emotion1)
+    print(emotion2)
+    print(f'MAE（評価データ）：{calculate_loss_and_accuracy(model, dataloader_train, device, criterion=criterion)[0]:.3f}')
+    print(f'QWK（評価データ）：{calculate_loss_and_accuracy(model, dataloader_dev, device, criterion=criterion)[0]:.3f}')
+    print(f'ACC（評価データ）：{calculate_loss_and_accuracy(model, dataloader_test, device, criterion=criterion)[0]:.3f}')
```
Comment (akiyama-ai) [inline fix_bert_twitter.py:206]: trainデータでMAEを測って、devデータでQWKを測って、testデータでACCを測ってることになってませんか？
テストデータで評価するなら、
mae, acc, qwk = calculate_loss_and_accuracy(model, dataloader_test, device, criterion=criterion)
と思います。
